Issue-71--updateSingleSpot

bmanzella opened the pull request:
Updated `put-spot.js` and `put-spot.spec.js` to make compatible with MongoDB.

#71

bmanzella committed:
Revert "Added my developer file"

This reverts commit 36efa46086d2507ac193ce04d98875b4a3a86817.

bmanzella committed:
Merge branch 'master' of https://github.com/raghunat/hotspot into issue-38--updateSpot

bmanzella committed:
Update files to make compatible with db

stephenthedev commented on spots-api/api/put-spot.js:
```diff
@@ -1,7 +1,11 @@
-    res.json(req.body);
+    var spots = db.collection('spots');
+    spots.update({_id: req.params.id}, {$set: req.body}, function(err, result) {
+      if(err) throw err;
```
change the if to:

```js
if(err) return res.status(500).json(code: 'DbError', err: err);
```

bmanzella committed:
Update put-spot.js

Modified if statement to `if(err) return res.status(500).json(code: 'DbError', err: err);`
